Synview-backend/prisma/migrations/20250706055003_added_invitations/migration.sql

```diff
@@ -0,0 +1,218 @@
+/*
+  Warnings:
+
+  - You are about to drop the `comment` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `goal` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `project` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `question` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `question_response` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `update` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `user` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `user_project` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- CreateEnum
+CREATE TYPE "Roles" AS ENUM ('USER', 'MENTOR');
+
+-- CreateEnum
+CREATE TYPE "invitation_status" AS ENUM ('PENDING', 'COMPLETE');
```

Comment:
Nit: I do not have a strong opinion on whether these should be camel case, snake case, etc. but for sure it should be consistent, ie. `roles` and `invitation_status` or `Roles` and `InvitationStatus`